app/src/pages/inside/settingsPage/settingsPage.jsx

```diff
@@ -1,10 +1,14 @@
+/* eslint no-unused-vars: 0,  spaced-comment: 0 */
```

Comment:
Do not change es-lint rules, even locally. In that case you can just comment line 6

Reply:
fixed